drivers/platform/aducm3029/irq.c

```diff
@@ -255,6 +290,30 @@ int32_t irq_unregister(struct irq_ctrl_desc *desc, uint32_t irq_id)
 						 NULL);
 		}
 		break;
+	case ADUCM_GPIO_A_INT_ID:
+		if (aducm_desc->conf[irq_id].gpio_conf) {
+			gpio_desc = aducm_desc->conf[irq_id].gpio_conf->gpio_handler;
+			if (!gpio_desc)
+				return FAILURE;
+			adi_gpio_RegisterCallback(ADI_GPIO_INTA_IRQ, NULL,
+						  NULL);
+			for (i = ADI_GPIO_PORT0; i < ADI_GPIO_NUM_PORTS; i++)
```

Comment:
I think adi_gpio_SetGroupInterruptPins(0xF, id, 0) should work

Reply:
My aim here is to unassign all pins assigned to on GPIO interrupt group. That means in my mind to get all ports. It doesn't seem that this suggestion accomplishes the same thing. Does it?

Reply:
My mistake, I didn't understand well adi_gpio_SetGroupInterruptPins functionality. It's ok how it is now.